### ISSUE TITLE
Help and Designer Settings Dialog issues

### ISSUE THREAD

**Issue by Epihaius** [fireclawthefox/DirectGuiDesigner#2]
Hi,

When editor scale is set to Pixel, the grid is drawn on top of the Designer Settings Dialog when the latter is displayed. This does not happen with the Help Dialog however.

A problem that occurs with both dialogs is that when the editor window is quite small, there does not seem to be a way to close these dialogs, as their OK/Cancel buttons are not accessible then.
If I may, I'd like to suggest assigning hotkeys to them for this purpose. For example, F1 might toggle the Help Dialog instead of just showing it, while Escape would be a good choice for closing both dialogs, I think.

It might also be a good idea to block the entire GUI whenever a dialog is shown, just to prevent bugs due to unforeseen user interaction; perhaps overlaying a window-filling MouseWatcherRegion with suppress flags on top of the GUI but below the dialog could work?

On a side note, I'm glad to see that the editor no longer starts in high resolution (1920 by 1080 pixels). This seemed to force the application to start in fullscreen mode and behaved very strangely on my laptop, to which I have a monitor attached whose native resolution is 1920 x 1080, while that of the laptop's built-in screen is a bit lower. What happened there was that the monitor blacked out for a split-second, then displayed the window in a slightly blurry, seemingly lower-res mode (although a screengrab I made was indeed 1920 x 1080), but the editor window was still decorated. There were some other applications open at that time, and when everything went back to normal after closing the editor, the windows of those applications had been downsized (presumably to fit that apparent lower resolution).

**Comment by fireclawthefox** [comment]
Hi,

thanks for the hint with the designer Settings dialog. I'll look into this.

As for the closing of the dialogs, I've already mapped F1 to toggle the help dialog on my local dev code. Will push that as soon as I got some more of the custom widgets stuff done. Regarding the GUI overlay to block stuff behind the dialog, yes, I guess that's the better way, I already have it for other dialogs, I just need to fix a bug that happens when resizing the window.

Last but not least, the resolution change, yeah, I did remove it as it was only for my setup anyways, sadly Panda3D doesn't offer a way to maximize windows at all, so it was the only way to maximize it on my systems.
